diff -up pypy-pypy-release-1.6/lib-python/2.7/test/test_subprocess.py.non-readable-path pypy-pypy-release-1.6/lib-python/2.7/test/test_subprocess.py
--- pypy-pypy-release-1.6/lib-python/2.7/test/test_subprocess.py.non-readable-path	2011-08-20 11:46:31.410646024 -0400
+++ pypy-pypy-release-1.6/lib-python/2.7/test/test_subprocess.py	2011-08-20 11:46:39.421645476 -0400
@@ -570,7 +570,7 @@ class ProcessTestCase(BaseTestCase):
         for i in range(1024):
             # Windows raises IOError.  Others raise OSError.
             with self.assertRaises(EnvironmentError) as c:
-                subprocess.Popen(['nonexisting_i_hope'],
+                subprocess.Popen(['/usr/bin/nonexisting_i_hope'],
                                  stdout=subprocess.PIPE,
                                  stderr=subprocess.PIPE)
             if c.exception.errno != errno.ENOENT:  # ignore "no such file"
